Error when running 'make'

**Describe the bug**
Error when running 'make'. 
```
Making all in .
make[1]: Entering directory '/home/ddmin/Downloads/mimic1-1.3.0.1'
  CC       main/mimic_main.o
  CC       main/mimic_lang_list.lo
  CC       src/audio/libttsmimic_la-au_alsa.lo
  CC       src/audio/libttsmimic_la-auclient.lo
  CC       src/audio/libttsmimic_la-au_command.lo
  CC       src/audio/libttsmimic_la-audio.lo
  CC       src/audio/libttsmimic_la-au_none.lo
  CC       src/audio/libttsmimic_la-au_oss.lo
  CC       src/audio/libttsmimic_la-au_portaudio.lo
  CC       src/audio/libttsmimic_la-au_pulseaudio.lo
  CC       src/audio/libttsmimic_la-auserver.lo
  CC       src/audio/libttsmimic_la-au_streaming.lo
  CC       src/audio/libttsmimic_la-au_sun.lo
  CC       src/audio/libttsmimic_la-au_wince.lo
  CC       src/cg/libttsmimic_la-cst_cg.lo
  CC       src/cg/libttsmimic_la-cst_mlsa.lo
  CC       src/cg/libttsmimic_la-cst_mlpg.lo
  CC       src/cg/libttsmimic_la-cst_vc.lo
  CC       src/cg/libttsmimic_la-cst_cg_load_voice.lo
  CC       src/cg/libttsmimic_la-cst_cg_dump_voice.lo
  CC       src/cg/libttsmimic_la-cst_cg_map.lo
  CC       src/cg/libttsmimic_la-cst_spamf0.lo
  CC       src/hrg/libttsmimic_la-cst_utterance.lo
  CC       src/hrg/libttsmimic_la-cst_relation.lo
  CC       src/hrg/libttsmimic_la-cst_item.lo
  CC       src/hrg/libttsmimic_la-cst_ffeature.lo
  CC       src/hrg/libttsmimic_la-cst_rel_io.lo
  CC       src/hts/libttsmimic_la-flite_hts_engine.lo
  CC       src/hts/hts_engine_API/lib/libttsmimic_la-HTS_audio.lo
  CC       src/hts/hts_engine_API/lib/libttsmimic_la-HTS_engine.lo
  CC       src/hts/hts_engine_API/lib/libttsmimic_la-HTS_misc.lo
  CC       src/hts/hts_engine_API/lib/libttsmimic_la-HTS_pstream.lo
  CC       src/hts/hts_engine_API/lib/libttsmimic_la-HTS_sstream.lo
  CC       src/hts/hts_engine_API/lib/libttsmimic_la-HTS_model.lo
In function 'HTS_Model_load_pdf',
    inlined from 'HTS_Model_load' at src/hts/hts_engine_API/lib/HTS_model.c:745:8:
src/hts/hts_engine_API/lib/HTS_model.c:701:7: error: 'free' called on pointer '*model.npdf' with nonzero offset 16 [-Werror=free-nonheap-object]
  701 |       free(model->npdf);
      |       ^~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[1]: *** [Makefile:3743: src/hts/hts_engine_API/lib/libttsmimic_la-HTS_model.lo] Error 1
make[1]: Leaving directory '/home/ddmin/Downloads/mimic1-1.3.0.1'
make: *** [Makefile:4809: all-recursive] Error 1
```

**To Reproduce**
Steps to reproduce the behavior:

```
1. $ sudo pacman -S --needed install gcc make pkg-config automake libtool alsa-lib
2. Download v1.3.0.1 release source code
3. $ cd mimic1
4. $ ./dependencies.sh --prefix="/usr/local"
5. $ ./autogen.sh
6. $ ./configure --prefix="/usr/local"
7. $ make
```

**Environment**
Linux 5.15.74-3-MANJARO

--- a/src/hts/hts_engine_API/lib/HTS_model.c
+++ b/src/hts/hts_engine_API/lib/HTS_model.c
@@ -698,7 +698,7 @@ static HTS_Boolean HTS_Model_load_pdf(HTS_Model * model, HTS_File * fp, size_t v
    }
    if (result == FALSE) {
       model->npdf += 2;
-      free(model->npdf);
+      HTS_free(model->npdf);
       HTS_Model_initialize(model);
       return FALSE;
    }

